code/modules/antagonists/clock_cult/items/clockwork_slab.dm

```diff
@@ -93,6 +93,18 @@ GLOBAL_LIST_INIT(clockwork_slabs, list())
 		calculated_cogs += difference
 		cogs += difference
 
+/obj/item/clockwork/clockwork_slab/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	switch(empowerment)
+		if("kindle")
+			kindle(user, target)
+		if("manacles")
+			hateful_manacles(user, target)
+		if("compromise")
+			sentinels_compromise(user, target)
+	if(active_scripture)
+		active_scripture.end_invokation()
```

Comment:
A much better way to do this would be to have some kind of active_scripture.on_attack() and then override the proc to add behaviour for specific spells.